ci/benchmark_ai.sh

```diff
@@ -0,0 +1,218 @@
+#!/bin/bash
+
+#Get option to run test on GPU/CPU image
+while getopts "g:" option; do
+  case $option in
+    g)      use_gpu=$OPTARG;;
+  esac
+done
+echo $use_gpu
+
+report=report.csv
+
+echo "$report"
+
+
+# Install Docker CLI in case it is missing
+install_docker(){
+    docker run --name hw hello-world >& /dev/null
+    if [[ `echo $?` != 0 ]] ;then
+       echo "Something went wrong with docker CLI installation"
+       # Install DOCKER CLI
+       apt-get remove docker docker-engine docker.io
+       apt-get update
+       apt-get install -y \
+            apt-transport-https \
+            ca-certificates \
+            curl \
+            software-properties-common
+       curl -fsSL https://download.docker.com/linux/ubuntu/gpg |  apt-key add -
+       apt-key fingerprint 0EBFCD88
+       add-apt-repository \
+	"deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+       apt-get update
+       apt-get install  -y docker-ce=18.03.1~ce-0~ubuntu
+    else
+        echo "Docker is happy"
+    fi
+}
+
+# Install Nvidia docker
+install_nvidia_docker(){
+    nvidia-docker --help  >& /dev/null
+    if [[ `echo $?` != 0 ]] ; then
+        curl -s -L https://nvidia.github.io/nvidia-docker/gpgkey | apt-key add -
+        distribution=$(. /etc/os-release;echo $ID$VERSION_ID)
+        curl -s -L https://nvidia.github.io/nvidia-docker/$distribution/nvidia-docker.list |tee /etc/apt/sources.list.d/nvidia-docker.list
+        apt-get update
+        apt-get install -y nvidia-docker2
+        pkill -SIGHUP dockerd
+    else
+        echo "nvidia docker already installed"
+    fi
+
+}
+
+
+
+
+pull_docker_images(){
+    # Pull correct docker images
+    if [ "$use_gpu" == "0" ]; then
+        docker pull awsdeeplearningteam/mms_cpu
+        if [[ `echo $?` != 0 ]] ; then
+            echo "ERROR: Exiting as docker pull failed for awsdeeplearningteam/mms_cpu"
+            exit 1
+        fi
+    elif [ "$use_gpu" == "1" ]; then
+        docker pull awsdeeplearningteam/mms_gpu
+        if [[ `echo $?` != 0 ]] ; then
+            echo "ERROR: Exiting as docker pull failed for awsdeeplearningteam/mms_cpu"
+            exit 1
+        fi
+    fi
+}
+
+#JMETER installation
+install_jmeter(){
+    echo "jmeter installation"
+    if [ -d "$HOME/apache-jmeter-4.0" ]; then
+        echo "JMeter already installed"
+    else
+        echo "Installing jmeter"
+        cd /tmp
```

Comment:
Should we append "&&" to the end of each command so that if one of the commands fails, the main function will get that error?? If one of the commands fails here, we will proceed with the rest of the commands right?